addons/stock_account/models/account_chart_template.py

```diff
@@ -31,5 +31,13 @@ def generate_properties(self, acc_template_ref, company, property_list=None):
             account = self[field]
             value = acc_template_ref[account].id if account else False
             PropertyObj._set_default(field, "product.category", value, company)
-
+        if self.property_stock_production_account_id:
+            production_account_id = acc_template_ref[self.property_stock_production_account_id].id if account else False
+            PropertyObj = PropertyObj.with_company(company)
+            property_stock_production = PropertyObj._get('property_stock_production', 'product.template')
+            if property_stock_production:
+                property_stock_production.write({
+                    'valuation_in_account_id': production_account_id,
+                    'valuation_out_account_id': production_account_id
+                })
```

Comment:
This looks redundant with what you're doing in `_compute_valuation_accounts`. Since all `production` locations are set with the account in the compute, do you still need to do this ?

Reply:
this location is created when the company is creating but at the same time, there are no COA loads so account is not set by compute method.
So when COA load for that company then we still need to do this.